src/extension/webView.ts

```diff
@@ -106,20 +106,24 @@ export async function createRepository(context: vscode.ExtensionContext) {
           }
 
           // Go to target folder & create app folder
-          const createAppFolder = `cd '${options.targetFolder}' && mkdir '${options.name}' && cd '${options.name}'`;
+          const createAppFolder = `mkdir '${options.targetFolder}${options.name}' -p && cd '${options.targetFolder}${options.name}'`;
           const openProject = `npm --no-git-tag-version version '${options.version}' && code .`;
           const installDependencies = options.nodeModules ? '--install' : '--no-install';
+          const legacyNodeVersions = ['12', '14'];
```

Comment:
What about version 11, 13 etc.? The used legacy node versions only apply in our CLI integration tests for Node.js - as we know the coupling from Node.js to npm.

Is `process` even doing what you want here? `process` would refer to `process` from VS Code, not from the Node process running in the command line.